examples/bilingual_document.ipynb

```diff
@@ -0,0 +1,305 @@
+    "deu_embeddings = text_embedding_pipeline.predict(deu_sentences, source_lang=\"deu_Latn\")\n",
+    "\n",
+    "eng_spa_cos_sim = cosine_similarity(eng_embeddings, spa_embeddings)\n",
+    "eng_deu_cos_sim = cosine_similarity(eng_embeddings, deu_embeddings)\n",
+    "spa_deu_cos_sim = cosine_similarity(spa_embeddings, deu_embeddings)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[ 0.72619224  0.08622358  0.06269339 ...  0.09275693  0.00628602\n",
+      "   0.22407693]\n",
+      " [ 0.11609402  0.46587974  0.05548527 ...  0.07190079 -0.00443545\n",
+      "   0.06152777]\n",
+      " [ 0.06567107  0.23206428  0.15597275 ...  0.06240445  0.15716389\n",
+      "   0.1589457 ]\n",
+      " ...\n",
+      " [ 0.04930421  0.13352732  0.09729137 ...  0.72410166  0.10511754\n",
+      "   0.09066909]\n",
+      " [ 0.02775934 -0.05379335  0.06206077 ...  0.12726557  0.7053859\n",
+      "   0.091729  ]\n",
+      " [ 0.2661551   0.11905408  0.12848586 ...  0.07357833  0.12886351\n",
+      "   0.67079294]]\n",
+      "[[ 0.682918    0.1001303   0.08201491 ...  0.10319082 -0.02233397\n",
+      "   0.19772843]\n",
+      " [ 0.14484891  0.4911242   0.13205954 ...  0.08779293  0.01014944\n",
+      "   0.03730423]\n",
+      " [ 0.08742855  0.2733525   0.17859717 ...  0.07594253  0.17223667\n",
+      "   0.14438803]\n",
+      " ...\n",
+      " [ 0.03904732  0.1515396   0.05158896 ...  0.6848761   0.03531618\n",
+      "   0.0910763 ]\n",
+      " [ 0.03424467 -0.04957096  0.0783025  ...  0.12372334  0.5359933\n",
+      "   0.08312893]\n",
+      " [ 0.26492468  0.14673     0.12761517 ...  0.04100559  0.10240456\n",
+      "   0.6001608 ]]\n",
+      "[[ 0.89043987  0.10906531  0.10327005 ...  0.06216208  0.04756077\n",
+      "   0.29304183]\n",
+      " [ 0.08295181  0.7090234   0.15835041 ...  0.15844154  0.01563987\n",
+      "   0.11715508]\n",
+      " [ 0.1153705   0.11315811  0.53616226 ...  0.04195565  0.1160554\n",
+      "   0.13952076]\n",
+      " ...\n",
+      " [ 0.11249126  0.14832962  0.06858915 ...  0.8037044   0.08767845\n",
+      "   0.0535988 ]\n",
+      " [ 0.0310357  -0.01143041  0.07956342 ...  0.0450792   0.67677176\n",
+      "   0.08205384]\n",
+      " [ 0.27339017  0.07933586  0.18632789 ...  0.06827298  0.13198397\n",
+      "   0.6999304 ]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(eng_deu_cos_sim)\n",
+    "print(eng_spa_cos_sim)\n",
+    "print(spa_deu_cos_sim)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Finding aligned sentence pairs\n",
+    "\n",
+    "We may find which of the sentences correspond to which in the translation by determining the cosine distance and considering equivalent in meaning those sentences that are the closest together in the embedding space."
```

Comment:
These comments are nice!
Could you please also explain that the peculiarity of DTW is that it is constrained to always find a monotonic alignment (a property that we want to hold for a bilingual text)?

Reply:
it would be nice to give short explanation and links to fastdtw.
Another nice feature is to show the optimal alignment path on cos-sim heatmap!

Reply:
+ show the cos-similarity of matching sentences (per pair of sentences and average)